ui/src/reducers/index.js

```diff
@@ -41,4 +41,5 @@ export default function createRootReducer(history) {
 export const REDUCERS_TO_PERSISTS = [
   { name: 'ui', initialState: uiInitialState },
   { name: 'user', initialState: userInitialState },
+  { name: 'search', initialState: searchInitialState },
```

Comment:
this will persist searches, does it work for references?

Reply:
Yes, references are defined in search states.